mne/dipole.py

```diff
@@ -268,6 +268,28 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             show_all, ax, block, show, scale=scale, color=color, fig=fig,
             title=title)
 
+    @verbose
+    def to_mni(self, subject, mri_head_t, subjects_dir=None,
+               verbose=None):
+        """Convert pos from head coordinate system to MNI ones.
+
+        Parameters
+        ----------
+        %(subject)s
+        mri_head_t : instance of Transform
+            MRI<->Head coordinate transformation.
```

Comment:
you should factorize the docstring as this is what we call `trans` everywhere. See fit_dipole function that writes:

```
    trans : str | None
        The head<->MRI transform filename. Must be provided unless BEM
        is a sphere model.
```

I would rename to `trans`

Reply:
Although I see that we have mri_head_t in head_to_mni function.

I think users are more used to the word `trans` in all tutorials etc.